src/hotspot/share/gc/z/zReferenceProcessor.cpp

```diff
@@ -125,9 +126,10 @@ void ZReferenceProcessor::set_soft_reference_policy(bool clear) {
   static LRUMaxHeapPolicy lru_max_heap_policy;
 
   if (clear) {
-    log_info(gc, ref)("Clearing All SoftReferences");
+    _always_clear_policy = true;
```

Comment:
I would prefer if this assignment was moved above the if and that you assign the field directly the value of the clear argument, instead of having two assignments.

Reply:
Fixed.